bundles/pubsub/pubsub_admin_tcp/src/pubsub_tcp_topic_sender.c

```diff
@@ -533,7 +529,8 @@ psa_tcp_topicPublicationSend(void *handle, unsigned int msgTypeId, const void *i
             clock_gettime(CLOCK_REALTIME, &serializationEnd);
         }
 
-        if (status == CELIX_SUCCESS /*ser ok*/) {
+        bool cont = pubsubInterceptorHandler_invokePreSend(sender->interceptorsHandler, entry->msgSer->msgName, msgTypeId, inMsg, &metadata);
```

Comment:
I'm not quite sure if we want to intercept messages if we can't serialize them. @pnoltes WDYT?

Maybe we should do short-circruiting here
```
bool cont = status == CELIX_SUCCESS && pubsubInterceptorHandler_invokePreSend(sender->interceptorsHandler, entry->msgSer->msgName, msgTypeId, inMsg, &metadata);
```

Reply:
I have changed this